clc/modules/object-storage-common/src/main/java/com/eucalyptus/objectstorage/exceptions/s3/S3Exceptions.groovy

```diff
@@ -197,6 +219,73 @@ class BucketNotEmptyException extends S3Exception {
   }
 }
 
```

Comment:
`S3Exceptions.groovy` is exclusively for S3 defined list of [error responses]( http://docs.aws.amazon.com/AmazonS3/latest/API/ErrorResponses.html#ErrorCodeList). Eucalyptus specific exceptions used internally for convenience should be defined in another class, probably under `com.eucalyptus.objectstorage.exceptions` package